tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
+        wsk.cli(Seq("property", "unset", "--auth", "--apihost", "--apiversion", "--namespace"))
+        var stdout = wsk.cli(Seq("property", "get", "--auth")).stdout
+        stdout should include regex ("""(?i)whisk auth\s*$""")      // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--apihost")).stdout
+        stdout should include regex ("""(?i)whisk API host\s*$""")  // default = empty string
+        stdout = wsk.cli(Seq("property", "get", "--namespace")).stdout
+        stdout should include regex ("""(?i)whisk namespace\s*_$""")// default = _
+        TestUtils.restoreWskProps(wskpropsBackup)
+        wskpropsBackup.delete()
```

Comment:
If the test fails anywhere before this, you will not restore the saved file. Another reason to avoid mucking around with it.